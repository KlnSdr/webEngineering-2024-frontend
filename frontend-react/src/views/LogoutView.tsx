function LogoutView() {
  return <div>Hello Logout</div>;
}

export default LogoutView;
