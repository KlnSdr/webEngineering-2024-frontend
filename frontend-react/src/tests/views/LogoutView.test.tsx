import React from "react";
import { render } from "@testing-library/react";
import LogoutView from "../../views/LogoutView";

describe("LogoutView Component", () => {
    it("matches the snapshot", () => {
        const { asFragment } = render(<LogoutView />);
        expect(asFragment()).toMatchSnapshot();
    });
});
