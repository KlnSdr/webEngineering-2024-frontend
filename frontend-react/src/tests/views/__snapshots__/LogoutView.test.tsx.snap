// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LogoutView Component matches the snapshot 1`] = `<DocumentFragment />`;
